tokio/src/task/error.rs

```diff
@@ -46,3 +47,15 @@ impl fmt::Debug for JoinError {
 }
 
 impl std::error::Error for JoinError {}
+
+impl From<JoinError> for io::Error {
```

Comment:
This conversion is done a bunch around the code base. Adding an explicit conversion seemed nice.